packages/x-charts/src/context/InteractionProvider.tsx

```diff
@@ -33,7 +33,13 @@ type InteractionActions<T extends ChartSeriesType = ChartSeriesType> =
     };
 
 type InteractionState = {
+  /**
+   * The item currently interacting.
+   */
```

Comment:
What is it interacting with? The user?

Reply:
Depends on how you trigger this provider :)